bin/quality

```diff
@@ -0,0 +1,59 @@
+#!/usr/bin/env ruby
+
+class Quality
+  require 'json'
+  require 'yaml'
+
+  MAX = 99
+  CELL_WIDTH = 10
+  attr_reader :cops_config
+
+  def initialize(config = '.quality.yml')
+    @cops_config = YAML.load_file(config)
+    @cops_config[:verbose] ||= false
+  end
+
+  def cops_everything
+    puts '  linter  |  alerts  |   max    |  status  '
```

Comment:
Before loop need use header like `------------------------` and footer for genering pseudo asci, check how it works with mysql explain